person.rb

```diff
@@ -0,0 +1,21 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'unknown', parent_permission: true)
+    @id = Random.random(1...1000)
```

Comment:
- [ Required ] Please consider changing this line of code to the one below because when I try to run the class I get an error in my console.

```ruby
  @id = Random.rand(1...1000)
```
![Screenshot from 2023-01-19 11-26-33](https://user-images.githubusercontent.com/16392046/213391719-8ac28f69-4de6-4556-892e-d51b770c7320.png)

Reply:
Thank you for this helpful change that you required, I have gone ahead to implement the change.